### ISSUE TITLE
Amending the colours used for text and possibly design elements on the website

### ISSUE THREAD

**Issue by SimonMurphyDI** [devinit/DIwebsite-redesign#1304]
**Is your feature request related to a problem? Please describe.**
The current website has some accessibility failures relating to the colour of text elements.

Tagging @edwinmp and @EmmawDI for their information. 

**Describe the solution you'd like**
We'd like to amend the colours so that the website meets WCAG AA or AAA standards. This document sets out the suggested changes 
https://docs.google.com/document/d/1NftD2UaK3ik2t298E5sY6xaR4PLVSfGkksHxk-K_FPs/edit?usp=sharing

**Links to pages**
I think this will affect most of the pages on the site - however I'm hopeful that (as it's simply changing hex codes?!) it might be relatively quick/easy!?

**Project**
We're getting a quote so we can ask the External relations directorate for budget to make the change 

**Tags**
Accessibility improvements  

**Budget Code**
As noted, will confirm once quote OKed. 

**Additional context**
I think the ideal would be to make the fewest changes to the website, so we're hoping we can change the text elements but not the design elements. My preference is that the text colours would be changed on the dev site, and we'd preview how that looks and then confirm it's just the text elements that need to change. Where text reverses out of a colour, changes will also be needed.

**Requested Delivery Date:**
It would be great to have a ballpark cost asap, as ideally we'd make this update in time for the GHA report launch.

**Comment by edwinmp** [comment]
Hi @SimonMurphyDI ... while the changes themselves are easy to make, there's a risk that the changes will leak where they shouldn't. Currently, the Pattern Library hasn't created a easy separation of colours for text, so we'll have to change a lot of files to get this done. It would be easier to change the brand colours completely (would take a few minutes really, an hour with testing). Otherwise, it'll take about a day for the text colour updates, with testing.

**Comment by SimonMurphyDI** [comment]
Thanks @edwinmp

**Comment by edwinmp** [comment]
Hi @SimonMurphyDI - apologies. I missed this. @davidebukali or @ThatcherK will be working on this.

**Comment by ThatcherK** [comment]
@SimonMurphyDI  this has been implemented please check staging to test the changes!

**Comment by SimonMurphyDI** [comment]
Thanks Thatcher. At the moment I can't see this being a priority, but it is important to keep track of it, as it would be good to do one day.
